app/src/main/java/org/oppia/android/app/devoptions/PlatformParameterRestartDialogFragment.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.android.app.devoptions
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import org.oppia.android.app.fragment.FragmentComponentImpl
+import org.oppia.android.app.fragment.InjectableDialogFragment
+import javax.inject.Inject
+
+/** Dialog fragment to restart the app when platform parameters are updated. */
```

Comment:
Considering reusability in different context (i.e. Force downloading).
```suggestion
/** Dialog fragment shown to prompt a full app restart. */
```

Reply:
updated